pretext/UnitTesting/WritingUnitTests.ptx

```diff
@@ -162,30 +162,7 @@ assert grade(92) == 'A'
 assert grade(85) == 'B'
 assert grade(69) == 'F'
         </input></program>
-                    .. reveal:: ac_grade_unittest_aux_reveal
```

Comment:
For pretext anything that we want instructors only to see like this should be wrapped in a `<commentary>` tag.

Reply:
I cannot find a <commentary> tag in the pretext documentation, where can I find that/ a tag with the function I need?

Reply:
See:  https://pretextbook.org/doc/guide/html/section-175.html